src/exchanges/subscription.ts

```diff
@@ -37,6 +37,8 @@ export interface ObservableLike<T> {
 }
 
 export interface SubscriptionOperation {
+  // TODO: why does this work?
```

Comment:
Note this here, somehow changing query to DocumentNode doesn't work since `print` expects an ASTNode